.buildkite/publish/dra/init_dra_publishing.sh

```diff
@@ -19,7 +19,18 @@ make clean sdist
 mkdir -p $RELEASE_DIR/dist/dra-artifacts
 cd -
 
-echo "The artifacts are: $(ls $RELEASE_DIR/dist)"
+# Download previous step artifacts
+buildkite-agent artifact download '.artifacts/*.tar.gz*' $RELEASE_DIR/dist/ --step build_docker_image_amd64
+buildkite-agent artifact download '.artifacts/*.tar.gz*' $RELEASE_DIR/dist/ --step build_docker_image_arm64
+cp $RELEASE_DIR/dist/.artifacts/* $RELEASE_DIR/dist/dra-artifacts
+
+# Rename to match DRA expectations (<name>-<version>-<classifier>-<os>-<arch>) (also, x86_64 instead of amd64)
+cd $RELEASE_DIR/dist/dra-artifacts
+mv elastic-connectors-docker-$VERSION-amd64.tar.gz elastic-connectors-$VERSION-docker-image-linux-x86_64.tar.gz
+mv elastic-connectors-docker-$VERSION-arm64.tar.gz elastic-connectors-$VERSION-docker-image-linux-arm64.tar.gz
```

Comment:
We could replace `elastic-connectors` with a reference to a new variable (maybe `$PROJECT_NAME`?).